src/mrinufft/operators/off_resonance.py

```diff
@@ -219,6 +219,7 @@ def __init__(
         r2star_map=None,
         B=None,
         tl=None,
+        negate=True,
```

Comment:
```suggestion
        isign=-1,
```

I feel its better to use `isign` which can be `-1` or `1`. Do make sure to check if it is either -1 or 1, else please raise an error

Reply:
@chaithyagr I tend to prefer the original way with a boolean. Here with an integer we should be checking that it's only -1 or 1, and we have to state it explicitly in the docstring while still adding more confusion. The boolean has the advantage of being fail safe, while the integer mostly discharge the developpers of half a line of code to add 3 more and put that charge on the user instead

Reply:
> Either `-1` (default, e^{-iωt}) or `1` (e^{iωt}).

That line in particular is confusing. Are we speaking about the forward operator or the inverse operator ? Inverse is used to get an image so it would be fair to assume one or the other

Reply:
Hmm, I think i agree with you... SOrry for the confusion @femaleprog , can you please revert it?